CMakeLists.txt

```diff
@@ -106,7 +110,7 @@ if (MSVC)
 endif()
 
 # If only clang-cpp is available, we take that.
```

Comment:
That comment is no longer relevant then.

Reply:
@aaronpuchert Ouch! Thanks for noticing, I'll clean that out.